WISE-Community/WISE
# Implement summary for scored components

Test out Open Response score summary
1. Log in as a teacher and open a project in the Authoring Tool
1. Create a Open Response step and author it with CRater item GREENROOF-II
1. Create a Summary step that shows the scores summary data for the previous Open Response step
1. Make sure the authoring preview for the Summary component shows sample summary data
1. Preview the project and make sure the Summary step shows sample summary data
1. Create a run with multiple periods with the project
1. Log in as student 1 and register to period 1 of the run and launch the run
1. Submit an answer for the Open Response (you can use the GREENROOF-II responses at the bottom of this comment), then go to the summary step to see the graph
1. Log in as student 2 and register to period 1 of the run and launch the run
1. Submit an answer for the Open Response and go to the summary step to see the graph
1. Log in as student 3 and register to period 2 of the run and launch the run
1. Submit an answer for the Open Response and go to the summary step to see the graph
1. Log in as the teacher
1. Launch the Classroom Monitor for the run
1. Go to the Grade By Step view and click on the Open Response step
1. Click on the Step Info button in the upper right and make sure the summary graph and data is correct
1. Change the period to period 2 and click the Step Info button and make sure it displays the correct summary data
1. Change the period to All Periods and click the Step Info button and make sure it displays the correct summary data

Other things to try out
- Set the max score in the Open Response to something besides 5 and then view the summary in the different places again (VLE, Authoring Tool, Classroom Monitor)

Test out Multiple Choice score summary
1. Add a Multiple Choice component to the project
2. Have a student submit work for the Multiple Choice
3. Have the teacher give that Multiple Choice work a score
4. View the Step Info for that Multiple Choice and make sure it shows the Responses Summary and the Scores Summary

**GREENROOF-II scores/responses**

1. I don't know.

2. The plants will taken in most of the heat from the sun to make its food, making your house cooler and so you don't have to crank the AC up.

3. The plant transform the energy from the sun into the energy that they need witch the energy is in the glucose and the energy mixes with the energy from the sun.

4. the sun gives the plants light energy. when it hits the plants it is used photosynthesis.the raw materials (water,and carbon) help  plants make sugars and glucose.

5. The sun rays would be absorbed by the plants and become chemical energy instead of hitting the roof and becoming heat energy that would then transfer into the house. The house may not need to use energy to cool down the house.

Closes #2005
